Applications/Games/STAR WARS - Empire at War - Gold Pack/Local/script.js

```diff
@@ -16,11 +16,11 @@ new LocalInstallerScript()
         new Downloader()
             .wizard(wizard)
             .url("http://static.dolimg.com/mh_netstorage/lucasfilm/patches/pc/EAW_RAM_MPLobby_update.exe")
-            .checksum("9fdecb9609a17f7358f2984a3278b484")
+            .checksum("63233107fab4c58413df04ce1d07fe65e7145329")
             .to(wine.prefixDirectory + "drive_c/users/Public/Documents/EAW_RAM_MPLobby_update.exe")
             .get();
         
-        wine.runInsidePrefix(wine.prefixDirectory + "drive_c/users/Public/Documents/EAW_RAM_MPLobby_update.exe","");
+        wine.run(wine.prefixDirectory + "drive_c/users/Public/Documents/EAW_RAM_MPLobby_update.exe","");
```

Comment:
I just checked again. `runInsidePrefix` is defined as `run(this.prefixDirectory + "/drive_c/" + executable, args)`. So in your case you get prefix + "/drive_c/" + prefix + "/drive_c/users...". If you write `runInsidePrefix("users/Public...")`, it should work.
Can you try that? I will merge this afterwards.